ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/PropertiesList.java

```diff
@@ -119,6 +118,11 @@
       FAIL_ON_DESERIALIZATION_ERROR_CONFIG
   );
 
+  /**
+   * List os properties that can be changes via `ALTER SYSTEM` command.
+   * We use this "allow list" for security reasons.
+   * (Independent of LD.)
```

Comment:
What's `LD` mean?

Reply:
LaunchDarkly (the tool that manages our feature flags which can be turned on/off at runtime in ccloud)